elements/lisk-codec/src/codec.ts

```diff
@@ -67,6 +67,15 @@ export const validateSchema = (schema: {
 		throw new LiskValidationError([...errors]);
 	}
 
+	const { properties } = schemaToValidate as ValidatedSchema;
+	if (properties) {
+		Object.values(properties).forEach(property => {
```

Comment:
Prefer to use `for...of` as it is slow although this only runs once so it would not make big difference